src/main/resources/application.yml

```diff
@@ -143,3 +143,6 @@ scheduler:
     fixedRate: PT10M
     initialDelay: PT10M
     shedlock-lock-at-most-for: PT10M
+  suspension:
+    cron: 0 */15 * * * *
```

Comment:
Might want to set this in config repo instead.

Reply:
This is the default. If you want to override it you can do it in the config. We have a default for all the others.